testsuite/mockserver/server.py

```diff
@@ -636,6 +636,38 @@ async def create_server(
         yield server
 
 
+@compat.asynccontextmanager
+async def create_unix_server(
+    socket_path: str,
+    loop,
+    testsuite_logger,
+    mockserver_reporter: reporter_plugin.MockserverReporterPlugin,
+    pytestconfig,
+) -> typing.AsyncGenerator[Server, None]:
+    async with net_utils.create_unix_server(
+        lambda: web_server(),
+        path=socket_path,
+    ):
+        mockserver_info = _create_unix_mockserver_info(socket_path)
```

Comment:
it's almost the same as create_server, please move common part to private function

Reply:
done